queries/kdl/folds.scm

```diff
@@ -0,0 +1,3 @@
+; Folds
+
+(node children: (node_children) @fold)
```

Comment:
Just `(node_children) @fold` would e equivalent?

Reply:
yep you're right, done